Getting Started Guide: update to reflect the new boot flow

gvancuts opened the pull request:
The boot process has changed a bit and the ACRN hypervisor is now
directly loaded by the platform EFI firmware. This commit reflects
those changes and also reference a newer version of Clearlinux that
includes the ACRN hypervisor that has these changes in.

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

gvancuts commented:
@dbkinder , how do you capture the png files that are in `getting_started/images`?

**Note:** if we wait about 24 hours, we could slightly simplify these and get rid of the following lines:
```
# cd /mnt/EFI/org.clearlinux/
# cp bootloaderx64.efi bootloaderx64_origin.efi
```

gvancuts commented:
@dbkinder , assuming you will test this, I'm curious to know where the `clr-boot-manager` commands will mess with what I set ealier (see `efibootmgr`). As you go through the steps, can you also capture the output of `efibootmgr -v` just after step 3 and then again after step 5?

I'll probably double-check that on my side too but it wouldn't hurt to have 2 pairs of eyes.

dbkinder commented on getting_started/index.rst:
```diff
@@ -128,33 +108,59 @@ partition. Follow these steps:
-      kernel-org.clearlinux.native.4.15.7-536
-      kernel-org.clearlinux.pk414-sos.4.14.23-19
-      kernel-org.clearlinux.pk414-standard.4.14.23-19
+      kernel-org.clearlinux.native.4.16.6-563
```
even with swupd off, these versions change (during installation it does a swupd)

dbkinder commented on getting_started/index.rst:
```diff
@@ -172,31 +178,27 @@ partition. Follow these steps:
-      match your set-up.
+   You will need to edit this file to adjust the kernel version (``linux`` section)
+   and also insert the ``PARTUUID`` of your ``/dev/sda3`` partition
+   (``root=PARTUUID=<><UUID of rootfs partition>``) in the ``options`` section.
```
This is a horrible usability problem -- running blkid to get the PARTUUID and copying it into the acrn.conf.  Yuck.

gvancuts replied:
It was my idea to replace the hard-coded `/dev/sda3` by a UUID... I think it's in general a better approach but I agree it's actually making things more awkward at the moment :(

gvancuts replied:
I believe the Clearlinux folks handle that automatically via the `clr-boot-manager` tool (they also use `PARTUUID` to indicate the root device).

gvancuts commented:
Hi @dbkinder , I have updated the PR and force-pushed the latest version.

Latest updates include:
* New pictures (that you sent me offline)
* New note about the fact that `swupd update` will change the EFI boot order if a kernel update was included
* Squashed all commits into one and added `Signed-off-by: David B. Kinder <david.b.kinder@intel.com>`

Thanks!

gvancuts commented:
@dbkinder can you take a look at the latest version of the patch?
